linter/lints/cpcps/lint_crl_has_idp.go

```diff
@@ -159,39 +158,43 @@ func parseSingleDistributionPointName(distributionPointName *cryptobyte.String,
 	fullNameTag := cryptobyte_asn1.Tag(0).ContextSpecific().Constructed()
 	if !distributionPointName.ReadASN1(distributionPointName, fullNameTag) {
 		return &lint.LintResult{
-			Status:  lint.Warn,
+			Status:  lint.Error,
 			Details: "Failed to read IssuingDistributionPoint distributionPoint fullName",
 		}
 	}
 
-	var uriBytes []byte
-	uriTag := cryptobyte_asn1.Tag(6).ContextSpecific()
-	if !distributionPointName.ReadASN1Bytes(&uriBytes, uriTag) {
-		return &lint.LintResult{
-			Status:  lint.Warn,
-			Details: "Failed to read IssuingDistributionPoint URI",
+	for !distributionPointName.Empty() {
```

Comment:
This function should be renamed `parseSingleDistributionPointName` -> `parseDistributionPointNames`.

Reply:
Weirdly the ASN.1 field is named "DistributionPointName" and there's only one of them inside the Issuing Distribution Point extension. But yeah this name is definitely misleading and I'll remove the "single" from it.